designsafe/apps/data/views/base.py

```diff
@@ -199,12 +199,27 @@ def get_context_data(self, **kwargs):
         elif pub.project.to_dict().get('doi') != None: #This is for older publications
             context['doi'] = pub.project.doi
         context['keywords'] = pub.project.value.keywords.split(',')
-        context['authors'] = [{
-            'full_name': '{last_name}, {first_name}'.format(
-                last_name=user['last_name'], first_name=user['first_name']
-            ),
-            'institution': getattr(getattr(user, 'profile'), 'institution', '')
-        } for user in getattr(pub, 'users', [])]
+        if pub.__contains__('users'):
+            context['authors'] = [{
+                'full_name': '{last_name}, {first_name}'.format(
+                    last_name=user['last_name'].encode('utf-8'), first_name=user['first_name'].encode('utf-8')
+                ),
+                'institution': getattr(getattr(user, 'profile'), 'institution', '')
+            } for user in getattr(pub, 'users', [])]
+        elif pub.__contains__('authors'):
+            context['authors'] = [{
+                'full_name': '{last_name}, {first_name}'.format(
+                    last_name=author['lname'].encode('utf-8'), first_name=author['fname'].encode('utf-8')
+                ),
+                'institution': getattr(author, 'inst', '')
+            } for author in getattr(pub, 'authors',[])]
+        else:
+            context['authors'] = [{
+                'full_name': '{last_name}, {first_name}'.format(
+                    last_name=author['lname'].encode('utf-8'), first_name=author['fname'].encode('utf-8')
+                ),
+                'institution': getattr(author, 'inst', '')
+            } for author in getattr(pub.project.value, 'teamOrder', [])]  
```

Comment:
Does `pub.project.value.teamOrder` work?

Reply:
`pub` on it's own is not a dictionary so it does not have the standard `get` methods until you do something like `pub.to_dict()`